book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Book
+  attr_accessor :title, :person
+
+  def initialize(title, person)
+    @title = title
+    person.books << self
```

Comment:
- [ ] Please make sure to create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. Example snippet:

```ruby

class Book
  
  # *Getters and Setters for @rentals should be present
  attr_accessor :title, :author, :rentals
  .
  .

  def initialize(title, author)
    .
    .
    # *@rentals should be initialized. 
    @rentals = []
  end
  
  # *A method for adding rentals should be present. Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
end

```